solutions/standard/main.tf

```diff
@@ -25,6 +25,33 @@ locals {
   parsed_existing_kms_instance_crn = var.existing_kms_instance_crn != null ? split(":", var.existing_kms_instance_crn) : []
   kms_region                       = length(local.parsed_existing_kms_instance_crn) > 0 ? local.parsed_existing_kms_instance_crn[5] : null
   existing_kms_guid                = length(local.parsed_existing_kms_instance_crn) > 0 ? local.parsed_existing_kms_instance_crn[7] : null
+  apply_cross_account_auth_policy  = !var.skip_kms_iam_authorization_policy && var.ibmcloud_kms_api_key != null ? 1 : 0
```

Comment:
Terraform creates resources. It is not applying an existing policy/resource to something. IMHO create_ is a better name.